discord/emoji.py

```diff
@@ -116,6 +116,14 @@ def mention(self) -> str:
             return f"<a:{self.name}:{self.id}>"
         return f"<:{self.name}:{self.id}>"
 
+    @property
+    def extension(self) -> str:
+        """Return the file extension of the emoji.
+
+        .. versionadded:: 2.8
```

Comment:
```suggestion
        .. versionadded:: 2.7.1
```

As discussed this is fine